src/m365/flow/commands/environment/environment-get.ts

```diff
@@ -40,22 +44,16 @@ class FlowEnvironmentGetCommand extends AzmgmtCommand {
     request
       .get(requestOptions)
       .then((res: any): void => {
-        if (args.options.output === 'text') {
-          logger.log({
-            name: res.name,
-            id: res.id,
-            location: res.location,
-            displayName: res.properties.displayName,
-            provisioningState: res.properties.provisioningState,
-            environmentSku: res.properties.environmentSku,
-            azureRegionHint: res.properties.azureRegionHint,
-            isDefault: res.properties.isDefault
-          });          
-        }
-        else {
-          logger.log(res);
-        }
+        res.name = res.name;
```

Comment:
We don't need to re-assign properties to the same names.

Reply:
Oops.. :(  Thanks for noticing